TODO.md

```diff
@@ -0,0 +1,25 @@
+## 9/8
```

Comment:
maybe this file should be "roadmap"? todos should be in issues. Idk, team thoughts?